packages/sdk/src/Base.ts

```diff
@@ -3,13 +3,13 @@ import memoize from 'fast-memoize'
 import { Addresses } from '@hop-protocol/core/addresses'
 import { BigNumber, BigNumberish, Contract, Signer, constants, providers } from 'ethers'
 import { Chain, Token as TokenModel } from './models'
-import { Chain as ChainEnum, MinPolygonGasPrice } from './constants'
+import { MinPolygonGasPrice, Slugs } from './constants'
 import { TChain, TProvider, TToken } from './types'
 import { config, metadata } from './config'
 import { getContractFactory, predeploys } from '@eth-optimism/contracts'
 import { parseEther, serializeTransaction } from 'ethers/lib/utils'
 
-export type ChainProviders = { [chain: string]: providers.Provider }
+export type ChainProviders = { [slug: string]: providers.Provider } | any
```

Comment:
why is `any` added here?

Reply:
thanks! it's from when i was fiddling around.